### ISSUE TITLE
Undocumented changes

### ISSUE THREAD

**Issue by RedDuckss** [Kinnay/NintendoClients#22]
There are a few differences between the source code of NintendoClients and the documentation which leads to certain undocumented differences

The ones which I have currently found deal with the type/flags and checksums

In https://github.com/Kinnay/NintendoClients/wiki/PRUDP-Protocol#v0-format it says that the "Type and flags" field is two bytes, however your source code shows a version which is one byte

https://github.com/Kinnay/NintendoClients/blob/a022b22d4d1bab16f6dd16f7679bd07546fd7e7b/nintendo/nex/prudp.py#L180-L183

Similarly the checksum is shown to only be one byte, but again your source code shows a version where there it is four bytes

https://github.com/Kinnay/NintendoClients/blob/a022b22d4d1bab16f6dd16f7679bd07546fd7e7b/nintendo/nex/prudp.py#L141-L160

From what I can tell these differing formats are not documented in the Wiki. There may be more as well, which I will leave here as comments if found, but at the moment I only have these 2

**Comment by Kinnay** [comment]
I already replied in private, but here's my opinion again:

As you might know, PRUDP was originally invented by Quazal. The 4-byte checksum and 1-byte flags are sometimes seen in games that use the original Quazal Rendez-Vous library, but I've never seen it in any 3DS, Wii U or Switch game.

Because the PRUDP protocol isn't easy to understand and the wiki page is already quite long I didn't want to add text that's unrelated to Nintendo's version of the protocol. Some people might still be interested though, so we tried to come up with a way to explain it in a way that's not confusing:

1. Describe the parts that are unrelated to Nintendo on a separate page. This would keep the page about the PRUDP protocol clean, but you'd have to move back and forth between the pages if you're working with non-Nintendo games.
2. Use a different color for the parts that are unrelated to Nintendo. This would make it easy to skip these parts if you only care about Nintendo games. Unfortunately this doesn't seem possible on github.
3. Add some kind of button that expands/collapses the parts that are unrelated to Nintendo. This seems to be possible with the `<details>` tag.

I'll probably play around with the third idea, and if it doesn't work out I'll go for the first idea.
